src/triggers/createPullRequestForJiraIssue.ts

```diff
@@ -106,7 +108,7 @@ export const createPullRequestForJiraIssue = async (
 
   info('Checking if there is an open pull request for this issue...')
   let pullRequestNumber
-  const repo = await getRepository(issue.fields.repository)
+  const repo = await getRepository(repository)
```

Comment:
@DanielM-SR can we make the repository param optional, and fall back on the `issue.fields.repository` as we used to? We still have 2 or 3 other Jira projects with the automation enabled, and I would like to keep backwards-compatibility if we can.